### PR TITLE
initial commit for SSH page

### PR DESCRIPTION
**Pull request by phredmund**
First try on the SSH page. Unfinished, but it's a start

### CODE REVIEW

File: src/content/docs/en/linux/ssh.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ssh [username@]<server url>
+```
+If you don't specify a username then SSH will attempt to connect you using the username with which you are logged in. Here are some examples:
+- `ssh myuser@myserver.com`
+    - This will attempt to connect to the server at myserver.com using the username myuser
+- `ssh myuser@192.168.1.2`
+    - This is just like the example above except you can also specify an IP address instead of a hostname
+- `ssh myserver.com`
+    - This will attempt to connect to the SSH server at myserver.com using your current username
+- `ssh 192.168.1.2`
```
Comment (nothingneko) [inline src/content/docs/en/linux/ssh.mdx:28]: the second IP example here is a tad redundant

**Review by nothingneko — COMMENTED** [review]
looks pretty good so far :3 
make sure you update consts.ts so this appears on the sidebar

keep up the good work

File: src/content/docs/en/linux/ssh.mdx (hunk 1 above)
Comment (phredmund) [reply src/content/docs/en/linux/ssh.mdx:28]: True. I went back and forth about including it, but figured more examples are better than fewer 🤷

File: src/content/docs/en/linux/ssh.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+title: SSH Secure Shell
+description: "An overview of the SSH utility."
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+Secure Shell ("SSH") is one of the most common utilities when access to a remote device is needed. It's encrypted, powerful, and simple to understand.
```
Comment (nothingneko) [inline src/content/docs/en/linux/ssh.mdx:8]: "easy" would make more sense than "simple"

File: src/content/docs/en/linux/ssh.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+title: SSH Secure Shell
+description: "An overview of the SSH utility."
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+Secure Shell ("SSH") is one of the most common utilities when access to a remote device is needed. It's encrypted, powerful, and simple to understand.
+
+## SSH overview
+SSH uses a client/server model where one computer acts as an SSH server[^1] and allows other computers to connect.
```
Comment (nothingneko) [inline src/content/docs/en/linux/ssh.mdx:11]: SSH uses a client/server model. One computer acts as a server[^1] and allows other computers (clients) to connect.

File: src/content/docs/en/linux/ssh.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    - This will attempt to connect to the SSH server at myserver.com using your current username
+
+When using any of the above commands, once connected to the remote server you'll be prompted for a password.
+
+## Running Your Own SSH Server
+Most modern Linux distros come with the required software to host your own SSH server. To allow users to connect to your machine, start the SSH server application using
+```zsh
+sudo systemctl start sshd
+```
+Note that the above command uses `sudo`. The `sudo` command will execute the following commands as root (the superuser). For many commands that involve starting and stopping servers, the permission of the system's superuser is necessary. You can read more about the `sudo` command [here](en/linux/permissions). The above command also uses the `systemctl` command. This command is mostly used to manage servers or other Linux services.
```
Comment (nothingneko) [inline src/content/docs/en/linux/ssh.mdx:36]: maybe use systemctl without sudo? this will invoke polkit which may be easier for users

**Review by nothingneko — APPROVED** [review]
looks pretty good. comments are pretty minor so we can just approve this now :3

**Pull request merged**